### DISCUSSION TITLE
Why does courseplay let my tractor drive in water?

### DISCUSSION THREAD

**Discussion by Jurgen1980** [Courseplay/Courseplay_FS22#2474] [Ideas]
Why does courseplay let my tractor drive in water?
This should never happen

**Comment by Tensuko** [comment]
Why I don't have any more information on this?

**Comment by Jurgen1980** [comment]
A tractor was unloading a combine and when he was driving from the grainmill(haut baileron) to the combine at field 33 he drove right in the pool between field 30 & 32.
I'm sorry for the minor info. I didn't take screenshots.

**Reply by Tensuko** [reply]
The Transport was done by Giants AI?

**Reply by Jurgen1980** [reply]
It was a 'CP:Unload combine' task.

**Reply by Tensuko** [reply]
Yes, but you said when he was driving from the grain mill, you used the Giants AI for driving back to the field.
The reason I ask is, where was the Targetposition set.

**Reply by Jurgen1980** [reply]
ah I see. The position was set on the dirt road just under the pool. I had no idea that part was done by the giants IA.

**Reply by Tensuko** [reply]
Well, if you don't use AutoDrive to do the transport, you can let Giants do it, but then ofc it will drive back to the Targetposition.
If that one is set in a bad spot, this pool time might happen :)
